src/org/zaproxy/zap/extension/spider/OptionsSpiderPanel.java

```diff
@@ -254,6 +267,13 @@ private JSlider getSliderThreads() {
 		}
 		return sliderThreads;
 	}
+	
+	private ZapNumberSpinner getDurationNumberSpinner() {
+		if (durationNumberSpinner == null) {
+			durationNumberSpinner = new ZapNumberSpinner(0, 60, Integer.MAX_VALUE);

```

Comment:
Maybe the default should be 0, to be consistent with the default of the persisted option?